.github/workflows/ci.yml

```diff
@@ -0,0 +1,70 @@
+on: [push, pull_request]
+name: CI
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        name:
+          [
+            btor,
+            cvc4,
+            msat,
+            yices2,
+            z3
```

Comment:
no bitwuzla?

Reply:
It's not public yet so there's no good way for me to pull it.